app/controllers/root_controller.rb

```diff
@@ -10,5 +10,6 @@ def index
                 else
                   []
                 end
+    @show_article_author = true
```

Comment:
I think now you should get rid of this controller because it does the same as ArticlesController